<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGALWindow</class>
 <widget class="QMainWindow" name="CGALWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1532</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDockWidget" name="infoDockWidget">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>370</y>
      <width>560</width>
      <height>240</height>
     </rect>
    </property>
    <property name="windowTitle">
     <string>Infos</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>536</width>
             <height>174</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="infoLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::DefaultContextMenu</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="displayLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="sceneDockWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>312</width>
      <height>302</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="windowTitle">
     <string>Geometric Objects</string>
    </property>
    <widget class="QWidget" name="dockWidgetContent">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="addButton">
            <property name="text">
             <string>+</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/cgal/icons/plus</normaloff>:/cgal/icons/plus</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="removeButton">
            <property name="text">
             <string>-</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/cgal/icons/minus</normaloff>:/cgal/icons/minus</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="duplicateButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/cgal/icons/duplicate</normaloff>:/cgal/icons/duplicate</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="searchEdit">
            <property name="placeholderText">
             <string>Quick filter... &lt;Alt+Q&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="sceneView">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="consoleDockWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>417</width>
      <height>240</height>
     </rect>
    </property>
    <property name="allowedAreas">
     <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::TopDockWidgetArea</set>
    </property>
    <property name="windowTitle">
     <string>Console</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="consoleTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="Viewer" name="viewer">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>639</width>
      <height>276</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1532</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuMesh">
    <property name="title">
     <string>Mesh</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuModel_Geometry">
    <property name="title">
     <string>Model Geometry</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPCL">
    <property name="title">
     <string>PCL</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMesh"/>
   <addaction name="menuModel_Geometry"/>
   <addaction name="menuPCL"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QOpenGLWidget</extends>
   <header>viewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
